code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -54,6 +54,9 @@
 	to_chat(user, "<span class='notice'>\The [src] is [anchored ? "":"not "]secured to the floor.</span>")
 
 /obj/structure/cult/functional/attackby(obj/I, mob/user, params)
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='danger'>\"you can seem to work the structure right with your bulky hands!.\"</span>")
```

Comment:
no capitalisation for the beginning word, and both a fullstop and an exclamation mark. The english is a bit dodge.

Also, it seems to say you CAN use it, but the immediate return shows you can't? Going to assume the can is an error also.

Suggest:

`Your bulky hands are inhibiting your ability to use this!`

or

`You cannot seem to manipulate this structure with your bulky hands!`

Something like that?